src/meta-srv/src/procedure/region_migration/downgrade_leader_region.rs

```diff
@@ -220,6 +236,61 @@ impl DowngradeLeaderRegion {
         }
     }
 
+    async fn update_leader_region_lease_deadline(&self, ctx: &mut Context) {
+        let leader = &ctx.persistent_ctx.from_peer;
+
+        let last_connection_at = match find_datanode_lease_value(leader.id, &ctx.in_memory).await {
+            Ok(lease_value) => lease_value.map(|lease_value| lease_value.timestamp_millis),
+            Err(err) => {
+                error!(err; "Failed to find datanode lease value for datanode: {}, during region migration, region: {}", leader, ctx.persistent_ctx.region_id);
+                return;
```

Comment:
If lease is not found, does it mean datanode is unreachable or metasrv is in some illegal state? Either way is simply log the err enough? Maybe we should deal with the error instead, for example, abort the procedure.